bin/node-debug.js

```diff
@@ -27,6 +27,10 @@ var argvOptions = {
     type: 'number',
     description: 'Node/V8 debugger port (`node --debug={port}`)'
   },
+  'debug-host': {
+    type: 'string',
+    description: 'Host where the debugged app is running (`node-inspector --debug-host={url}`)'
+  },
```

Comment:
This doesn't make much sense to me, to be honest. `node-inspect` is intended to be run on the same machine where you 1) run the debugged app 2) open the browser. When debugging remotely, you have to start the debugged app manually. After that, there is little use in `node-inspect`, one can directly start `node-inspector`.

I am proposing to revert changes in this file (`bin/node-debug.js`).

If you disagree, then please check the output of `npm test`, there are several test failures caused by these changes.

https://travis-ci.org/node-inspector/node-inspector/jobs/183141504
```
 2) node-debug argument parser handles `app.js`:
      AssertionError: expected { Object (printScript, subproc, ...) } to deeply equal { Object (printScript, subproc, ...) }
      + expected - actual
       {
         "inspector": {
           "args": [
             "--web-port=8080"
      -      "--debug-host="
           ]
      -    "debugHost": ""
           "port": 8080
         }
         "printScript": true
         "subproc": {
      
      at Context.<anonymous> (test/node-debug.test.js:9:25)
  3) node-debug argument parser handles options:
      AssertionError: expected { Object (port, debugHost, ...) } to deeply equal { port: 10, args: [ '--web-port=10' ] }
      + expected - actual
       {
         "args": [
           "--web-port=10"
      -    "--debug-host="
         ]
      -  "debugHost": ""
         "port": 10
       }
      
      at Context.<anonymous> (test/node-debug.test.js:39:35)
  4) node-debug argument parser forwards unknown options to node-inspector:
      inspector args
      + expected - actual
         "--no-some-other-bool"
         "--some-string"
         "val"
         "--web-port=8080"
      -  "--debug-host="
       ]
      
      at Context.<anonymous> (test/node-debug.test.js:82:13)
```